File: 1. Learn the Basics/1. Pattern/Python/16.py
# A 
# B B 
# C C C 
# D D D D 
# E E E E E 

def alphaRamp(n):
    letter = 65
    for i in range(0,n):
        for j in range(0,i+1):
            print(chr(letter),end=" ")
        letter = letter+1
        print("\r")

        
n = int(input())
alphaRamp(n)

File: 1. Learn the Basics/1. Pattern/Java/16.java
/* package codechef; // don't place package name! */

/*

A 
B B 
C C C 
D D D D 
E E E E E 

*/
import java.util.*;
import java.lang.*;
import java.io.*;


/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
    public void alphaRamp(int n) {
        // Write your code here
        char letter = 'A';
        for(int i=0;i<n;i++){
            for(int j=0;j<=i;j++){
                System.out.print(letter+" ");
            }
            System.out.println();
            letter++;
        }
    }
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Codechef obj = new Codechef();
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		obj.alphaRamp(n);
	}
}
